tsconfig.browser.json

```diff
@@ -5,7 +5,10 @@
     "module": "esnext",
   },
   "include": [
-    "src/**/public/**/*"
+    // in the build we populate this to include **/public/**/* but
```

Comment:
is the config loader okay with comments in json?  i don't think this will lint

Reply:
Yeah, typescript config can have comments, checkout `tsconfig.json`. It has a bunch of them and always has.